scripts/build_run.sh

```diff
@@ -4,6 +4,9 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
+export CXX=icpx
+export CC=icx
+
 cmake -B build -DENABLE_SYCL=on
```

Comment:
ENABLE_SYCL has no effect in project's CMakeList.txt.
The examples require SYCL compiler to be available, not enabling SYCL makes no sense. Consider adding a user friendly message about missing SYCL compiler, if icpx is not available (in CMakeList.txt probably).

Reply:
The script was left only to make it easier to run all with one command, I should probably make another pr with it, let's maybe leave it as it is (with no flag - thx) and I will enhance it's functionality in another PR ;)